components/radio/src/index.js

```diff
@@ -84,25 +84,39 @@ const LabelText = styled('span')({
 });
 
 const Radio = ({
-  inline, children, className, disabled, ...input
+  inline, children, disabled, checked, name, value, input, ...props
```

Comment:
i'm not sure about these changes

this component is now expecting an `input` object in it's props for values to pass through to the `input` element

it's probably wiser to keep this component as it was